js/src/api/format/output.js

```diff
@@ -129,12 +129,31 @@ export function outNumber (number) {
   return new BigNumber(number || 0);
 }
 
+export function outPeer (peer) {
```

Comment:
Would really have liked tests for this. (API is close to full coverage, not for the sake of coverage, but rather since it is such a critical piece.)